src/resources.scala

```diff
@@ -487,9 +491,11 @@ object Resources {
 
           IO.write(tr, trTemplate format (p,
             if (withViewHolders) "" else  " extends AnyVal",
-            resources map { case (k,v) =>
-              "  final val %s = TypedResource[%s](R.id.%s)" format (wrap(k),v,wrap(k))
-            } mkString "\n",
+            if (ids) {
+              resources map { case (k,v) =>
+                "  final val %s = TypedResource[%s](R.id.%s)" format (wrap(k),v,wrap(k))
+              } mkString "\n"
+            } else "  // Id generation disabled via typedResourcesIds := false",
```

Comment:
I would like it to say `TypedResource ID generation disabled by 'typedResourcesIds := false'`